src/main/java/net/masterthought/cucumber/json/Step.java

```diff
@@ -63,6 +63,9 @@ public Result getResult() {
     public boolean hasRows() {
         return ArrayUtils.isNotEmpty(rows);
     }
+    public boolean hasDocString() {
+        return doc_string!=null;
```

Comment:
`doc_string` is not an array so `ArrayUtils.isNotEmpty` would not work.

How about:  `doc_string != null && StringUtils.isNotEmpty(doc_string.getValue())` ?